server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -221,15 +252,6 @@ export class ScribeLambda implements IPartitionLambda {
                                     this.updateProtocolHead(this.protocolHandler.sequenceNumber);
                                     this.updateLastSummarySequenceNumber(this.protocolHandler.sequenceNumber);
                                     const summaryResult = `Client summary success @${value.operation.sequenceNumber}`;
-                                    this.context.log?.info(
```

Comment:
Any reason we are removing this?

Reply:
I restored this in addition to the new lumberjack logs